components/places/src/frecency.rs

```diff
@@ -150,7 +151,7 @@ impl<'db, 's> FrecencyComputation<'db, 's> {
         ",
         )?;
         let mut rows = stmt.query(&[(":page_id", &page_id)])?;
-        let row = rows.next()?.ok_or(rusqlite::Error::QueryReturnedNoRows)?;
+        let row = trace_error!(rows.next()?.ok_or(rusqlite::Error::QueryReturnedNoRows))?;
```

Comment:
and ditto here with the same link? I'm really just thinking about how someone in the future can undo the various parts used to track the specific problem.

(Not too sure about the breadcrumbs below - I guess probably not? It could be argued they are useful in the general case, and it seems reasonable that we could just remove breadcrumbs later if we feel they haven't added value recently?)

Reply:
That makes a lot of sense.  I added comments to the frecency breadcrumbs too, I kind of doubt they are going to be useful in general.